### PR TITLE
add channels.replies facet to web api client

### LINKED ISSUE #378: Missing web.channels.replies() facet?
Doesn't seem to be listed in here anywhere: https://github.com/slackapi/node-slack-sdk/blob/master/lib/clients/web/facets/channels.js

Should be simple/easy enough to implement, any idea why this hasn't already been done?

### PR DESCRIPTION
**Pull request by mismith**
* [x] I've read and understood the [Contributing guidelines](./CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](./CODE_OF_CONDUCT.md).
* [x] I've been mindful about doing atomic commits, adding documentation to my changes, not refactoring too much.
* [x] I've a descriptive title and added any useful information for the reviewer. Where appropriate, I've attached a screenshot and/or screencast (gif preferrably).
* [ ] I've written tests to cover the new code and functionality included in this PR.
* [x] I've read, agree to, and signed the [Contributor License Agreement (CLA)](https://docs.google.com/a/slack-corp.com/forms/d/1q_w8rlJG_x_xJOoSUMNl7R35rkpA7N6pUkKhfHHMD9c/viewform).

#### PR Summary
I simply added a missing facet, copying over what was appropriate from the official docs here: https://api.slack.com/methods/channels.replies

#### Related Issues
Fixes #378.

#### Test strategy
No tests were added since the existing tests don't target specific facets or API calls (hence why this omission wasn't caught in the first place).

### CODE REVIEW

**Comment by Roach** [comment]
@mismith thanks for adding this 👍 

Would you mind signing the CLA (linked in the comments above) so we can get this merged?

**Comment by mismith** [comment]
I have signed both :)

**Comment by Roach** [comment]
@mismith ah, I see the problem. You need to add that email to your github account: https://github.com/settings/emails

If you look at the commit log, you can see that it's associated with the email and not your github account: https://github.com/slackapi/node-slack-sdk/pull/380/commits/d4b7438bf997f15f4cadbd832b3b763b91c574a0

File: lib/clients/web/facets/channels.js [hunk 1]
```diff
@@ -214,6 +215,24 @@ ChannelsFacet.prototype.rename = function rename(channel, name, optCb) {
...
 
 /**
+ * Retrieve a thread of messages posted to a channel.
+ * @see {@link https://api.slack.com/methods/channels.replies|channels.replies}
+ *
+ * @param {?} channel - Channel to fetch thread from
+ * @param {?} thread_ts - Unique identifier of a thread's parent message
+ * @param {function=} optCb Optional callback, if not using promises.
+ */
+ChannelsFacet.prototype.replies = function rename(channel, thread_ts, optCb) {
```
Comment (aoberoi) [inline lib/clients/web/facets/channels.js]: please change `thread_ts` to `threadTs` (above in the JSDoc too)

**Comment by mismith** [comment]
Sorry I no longer have time to work on this :(

**Pull request closed without merging**